tests/unit/lib/foxbox/api_test.js

```diff
@@ -2,7 +2,7 @@ import { waitForNextMacroTask } from '../../test-utils';
 import API from 'js/lib/foxbox/api';
 
 describe('API >', function () {
```

Comment:
It'd be nice to start adding the annotations for tests when we modify unit tests files (`/** @test {API} */`).
The documentation generator will land soon.